README.md

```diff
@@ -163,15 +173,21 @@ Visit http://localhost:8000 on your browser to access the API documentation.
 
 ## Setup Frontend
 
-- Create a file named **.env** in the frontend root folder (same directory with package.json)
-- Copy the content of **.env.example** and paste into the new .env file.
 - On your terminal/command line, navigate to **./zubhub/zubhub_frontend/zubhub** directory
 
+- Create a file named **.env** in the frontend root folder (same directory with package.json), then copy-past the following lines
+
+```sh
+#.env
+REACT_APP_NODE_ENV=developement
+REACT_APP_BACKEND_DEVELOPMENT_URL=http://127.0.0.1:8000
+```
+
 ### **Using npm**
 
 This is advisable for a better development experience.
 
-- Run `npm install` to install the dependencies.
+- Run `npm install --legacy-peer-deps` to install the dependencies.
```

Comment:
Can you elaborate why we need this dependency?

Reply:
The --legacy-peer-deps flag was introduced with v7 as a way to bypass peerDependency auto-installation.  Due to the large number of modules that haven't specifically added React v17 as a peerDependency, it's now commonplace to encounter the unable to resolve dependency tree error when running npm installs within a v17 React application. So adding this flag will solve the issue.